WeakAuras/Types.lua

```diff
@@ -3801,6 +3806,8 @@ if WeakAuras.IsClassic() then
   wipe(Private.multiUnitUnits.boss)
   wipe(Private.multiUnitUnits.arena)
   Private.unit_types.focus = nil
+  Private.unit_types_bufftrigger_2.softenemy = nil
+  Private.unit_types_bufftrigger_2.softfriend = nil
```

Comment:
This was jsut based on the first commit. Since you added softenemy/softfriend to more, it needs to be removed from more for Classic